Resources/Prototypes/Loadouts/Generic/species.yml

```
@@ -50,3 +50,28 @@
+  id: LoadoutSpeciesMantleMag
+  category: Species
+  cost: 6
+  requirements:
+    - !type:CharacterSpeciesRequirement
+      species:
+        - Harpy
+        - Diona
+        - Arachne
+    - !type:CharacterLogicOrRequirement
+      requirements:
+      - !type:CharacterDepartmentRequirement
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)